src/libs/actions/Report.js

```diff
@@ -674,30 +647,53 @@ function fetchActions(reportID, offset) {
  *
  * @param {Boolean} shouldRedirectToReport this is set to false when the network reconnect code runs
  * @param {Boolean} shouldRecordHomePageTiming whether or not performance timing should be measured
+ * @param {Number} fetchActionsDelay
```

Comment:
description?

Reply:
if the others didn't have descriptions would you have noticed ? 😄

Reply:
Dunno - do you think a description would be worthwhile or not?